ci/docker-compose-tests.yaml

```diff
@@ -23,11 +23,8 @@ services:
         - http_proxy
         - https_proxy
         - no_proxy
-    volumes:
-      - ../:/project/TrustedComputeFramework
-    working_dir: "/project/TrustedComputeFramework/tools/build"
+    working_dir: "/project/TrustedComputeFramework/"
```

Comment:
WORKDIR is set in Dockerfile. Do we still need to set this in docker-compose file. Since the build is done from Dockerfile, Can we remove this line in docker-compose file ? I notices that "working_dir" is different for ci/docker-compose-test.yaml and docker-compose.yaml. Is that intentional ?